notebooks/GPU.ipynb

```diff
@@ -1,14 +1,20 @@
 {
   "cells": [
     {
-      "cell_type": "markdown",
+      "cell_type": "code",
+      "source": [
+        "# Taken from KoboldAI's Colab.\n",
+        "# If it ain't broke don't fix it lol\n",
+        "#@title <-- Tap this if you play on Mobile { display-mode: \"form\" }\n",
+        "%%html\n",
+        "<b>Press play on the music player to keep the tab alive, then start KoboldAI below (Uses only 13MB of data)</b><br/>\n",
```

Comment:
Huh. Yeah, I probably should do that too. I figured since it's not actually visible on Colab, no one will see it, but it probably does show up in some way.